specification/commerce/resource-manager/Microsoft.Commerce/preview/2015-06-01-preview/commerce.json

```diff
@@ -151,6 +172,14 @@
   "definitions": {
     "InfoField": {
       "properties": {
+        "meteredRegion": {
```

Comment:
these are new properties.
Need to be a new api-version

Reply:
These are actually existing properties that were incorrectly missing in the Swagger document. We did not update the api, so a new api version is not required.

These properties can be seen in the first sample response for the current api version:
https://learn.microsoft.com/en-us/previous-versions/azure/reference/mt219001(v=azure.100)?redirectedfrom=MSDN#response-codes

Reply:
Just to be explicitly clear. Our team did not in any way update the way that the Api works. The swagger document is being updated to match its current behavior.

Reply:
Upon further review, it looks like our API returns a blank response for Infofields, so these properties are no longer used. I will remove them from the example and Commerce.json.